File: common/src/main/java/com/hartwig/actin/molecular/datamodel/evidence/Country.java
package com.hartwig.actin.molecular.datamodel.evidence;

import com.hartwig.actin.Displayable;

import org.jetbrains.annotations.NotNull;

public enum Country implements Displayable {
    NETHERLANDS("Netherlands"),
    BELGIUM("Belgium"),
    GERMANY("Germany"),
    OTHER("Other");

    @NotNull
    private final String display;

    Country(@NotNull final String display) {
        this.display = display;
    }

    @Override
    @NotNull
    public String display() {
        return display;
    }
}

File: common/src/main/kotlin/com/hartwig/actin/molecular/datamodel/evidence/Country.kt
package com.hartwig.actin.molecular.datamodel.evidence

import com.hartwig.actin.Displayable

enum class Country(private val display: String) : Displayable {
    NETHERLANDS("Netherlands"),
    BELGIUM("Belgium"),
    GERMANY("Germany"),
    US("United States"),
    OTHER("Other");

    override fun display(): String {
        return display
    }
}
